ipi/engine/forcefields.py

```diff
@@ -446,6 +447,114 @@ def evaluate(self, r):
         r["status"] = "Done"
 
 
+class FFdmd(ForceField):
+
+    """Basic fully pythonic force provider.
+
+    Computes DMD forces as in Bowman, .., Brown JCP 2003. It is a time dependent potential.
```

Comment:
Can you provide a full citation ?  and describe the potential in a bit more detail ?